README.md

```diff
@@ -1,78 +1,43 @@
-
-1. [Kubernetes](Documentation/kubernetes.md) (recommended)
-2. [Standalone](Documentation/standalone.md)
-
-### Using Rook
-
-Once you have a Rook cluster running, you can use the `rookctl` tool to create and manage storage as shown in the following guide:
-- [Using Rook Guide](Documentation/client.md)
+Rook is an open source orchestrator for distributed storage systems running in cloud native environments.
 
-## Advanced Configuration and Troubleshooting
+Rook turns distributed storage software into a self-managing, self-scaling, self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management by utilizing the underlying
+cloud-native container management, scheduling and/or orchestration platform.
 
-Our Rook toolbox container is available to aid with troubleshooting and advanced configuration of your Rook cluster.
-It automatically configures a Ceph client suite to work with your Rook deployment, and additional tools are just an `apt-get` away.
+Rook integrates deeply into cloud native environments leveraging extension points and providing a seamless experience for scheduling, lifecycle management, resource management, security, monitoring, and user experience.
 
-To get started please see the [toolbox readme](Documentation/toolbox.md).  Also see our [advanced configuration](Documentation/advanced-configuration.md) document for helpful maintenance and tuning examples.
+Rook is currently in alpha state and has focused initially on orchestrating Ceph on-top of Kubernetes. Ceph is a distributed storage system that provides file, block and object storage and is deployed in large scale production clusters. Rook plans to add support for other storage systems beyond Ceph and other cloud native environments beyond Kubernetes in future releases. See our [roadmap](ROADMAP.md) for more details.
 
-## Building
+## Getting Started and Documentation
 
-See [Building](https://github.com/rook/rook/wiki/Building) in the wiki for more details.
+For installation, deployment, and administration, see our [Documentation](https://rook.github.io/docs/rook/latest/kubernetes.html).
 
 ## Contributing
 
 We welcome contributions. See [Contributing](CONTRIBUTING.md) to get started.
 
 ## Report a Bug
 
-For filing bugs, suggesting improvements, or requesting new features, help us out by opening an [issue](https://github.com/rook/rook/issues).
+For filing bugs, suggesting improvements, or requesting new features, please open an [issue](https://github.com/rook/rook/issues).
 
 ## Contact
 
 Please use the following to reach members of the community:
 
-- Email: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
+- Forums: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
```

Comment:
should Gitter be first in this list, due to its overwhelming popularity compared to the google forums?